wandb/sdk/artifacts/artifact.py

```diff
@@ -2316,63 +2276,10 @@ def _normalize_metadata(metadata: dict[str, Any] | None) -> dict[str, Any]:
             Dict[str, Any], json.loads(json.dumps(util.json_friendly_val(metadata)))
         )
 
-    def _load_manifest(self, url: str) -> None:
-        with requests.get(url) as request:
-            request.raise_for_status()
-            self._manifest = ArtifactManifest.from_manifest_json(
-                json.loads(util.ensure_text(request.content))
-            )
-
-    @staticmethod
-    def _get_gql_artifact_fragment() -> str:
-        fields = InternalApi().server_artifact_introspection()
-        fragment = """
-            fragment ArtifactFragment on Artifact {
-                id
-                artifactSequence {
-                    project {
-                        entityName
-                        name
-                    }
-                    name
-                }
-                versionIndex
-                artifactType {
-                    name
-                }
-                description
-                metadata
-                ttlDurationSeconds
-                ttlIsInherited
-                aliases {
-                    artifactCollection {
-                        project {
-                            entityName
-                            name
-                        }
-                        name
-                    }
-                    alias
-                }
-                _MAYBE_TAGS_
-                state
-                commitHash
-                fileCount
-                createdAt
-                updatedAt
-            }
-        """
-        if "ttlIsInherited" not in fields:
-            fragment = fragment.replace("ttlDurationSeconds", "").replace(
-                "ttlIsInherited", ""
-            )
-
-        if "tags" in fields:
-            fragment = fragment.replace("_MAYBE_TAGS_", "tags {name}")
-        else:
-            fragment = fragment.replace("_MAYBE_TAGS_", "")
-
-        return fragment
+    def _load_manifest(self, url: str) -> ArtifactManifest:
```

Comment:
note: now returns an `ArtifactManifest` for clarity, rather than assigning in-place